randovania/games/dread/logic_database/Ferenia.txt

```diff
@@ -800,10 +800,16 @@ Extra - asset_id: collision_camera_012
       All of the following:
           Morph Ball
           Any of the following:
-              Grapple Beam
+              Spider Magnet
               All of the following:
                   Gravity Suit
-                  Space Jump or Simple IBJ
+                  Any of the following:
+                      Spin Boost or Simple IBJ
+                      # Slide Jump - https://youtu.be/3mfzhqqZBlc
+                      Single-wall Wall Jump (Beginner) and Slide Jump (Beginner)
+                      # Beginner Wall Jump - https://youtu.be/iHFyp9ON-pM
+                      Flash Shift and Wall Jump (Beginner)
```

Comment:
I'm not necessarily opposed to having the wall jump on the flash shift here, but I feel like I should point out that it appears common practice to have a single wall jump combined with a flash shift as trickless in a bunch of places, including going to the left ledge in this room.